notebooks/Overview.ipynb

```diff
@@ -1,5621 +1,5149 @@
-      " 'xor_tydi_qa', 'xquad', 'xquad_r', 'xsum', 'xsum_factuality', 'xtreme',\n",
-      " 'yahoo_answers_qa', 'yahoo_answers_topics', 'yelp_polarity',\n",
-      " 'yelp_review_full', 'yoruba_bbc_topics', 'yoruba_gv_ner', 'yoruba_text_c3',\n",
-      " 'yoruba_wordsim353', 'youtube_caption_corrections', 'zest',\n",
-      " 'JesseParvess/Book_Snippets_ASR']\n",
-      "🤩 Currently 33 metrics are available on the hub:\n",
-      "['accuracy', 'bertscore', 'bleu', 'bleurt', 'cer', 'chrf', 'code_eval', 'comet',\n",
-      " 'competition_math', 'coval', 'cuad', 'f1', 'gleu', 'glue', 'google_bleu',\n",
-      " 'indic_glue', 'matthews_correlation', 'meteor', 'pearsonr', 'precision',\n",
-      " 'recall', 'rouge', 'sacrebleu', 'sari', 'seqeval', 'spearmanr', 'squad',\n",
-      " 'squad_v2', 'super_glue', 'ter', 'wer', 'wiki_split', 'xnli']\n"
-     ]
-    }
-   ],
-   "source": [
-    "# Currently available datasets and metrics\n",
-    "datasets = list_datasets()\n",
-    "metrics = list_metrics()\n",
-    "\n",
-    "print(f\"🤩 Currently {len(datasets)} datasets are available on the hub:\")\n",
-    "pprint(datasets, compact=True)\n",
-    "print(f\"🤩 Currently {len(metrics)} metrics are available on the hub:\")\n",
-    "pprint(metrics, compact=True)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 4,
-   "metadata": {
-    "colab": {
-     "base_uri": "https://localhost:8080/",
-     "height": 544
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "zNp6kK7OvSUg",
+        "pycharm": {
+          "name": "#%% md\n"
+        }
+      },
+      "source": [
+        "# HuggingFace 🤗 Datasets library - Quick overview\n",
+        "\n",
+        "Models come and go (linear models, LSTM, Transformers, ...) but two core elements have consistently been the beating heart of Natural Language Processing: Datasets & Metrics\n",
+        "\n",
+        "🤗 Datasets is a fast and efficient library to easily share and load dataset and evaluation metrics, already providing access to 150+ datasets and 12+ evaluation metrics.\n",
+        "\n",
+        "The library has several interesting features (beside easy access to datasets/metrics):\n",
+        "\n",
+        "- Build-in interoperability with PyTorch, Tensorflow 2, Pandas and Numpy\n",
+        "- Lighweight and fast library with a transparent and pythonic API\n",
+        "- Strive on large datasets: frees you from RAM memory limits, all datasets are memory-mapped on drive by default.\n",
+        "- Smart caching with an intelligent `tf.data`-like cache: never wait for your data to process several times\n",
+        "\n",
+        "🤗 Datasets originated from a fork of the awesome Tensorflow-Datasets and the HuggingFace team want to deeply thank the team behind this amazing library and user API. We have tried to keep a layer of compatibility with `tfds` and a conversion can provide conversion from one format to the other."
```

Comment:
Maybe adding a reference to the Evaluate library, as you talked about metrics above:
```suggestion
        "🤗 Datasets originated from a fork of the awesome Tensorflow-Datasets and the HuggingFace team want to deeply thank the team behind this amazing library and user API. We have tried to keep a layer of compatibility with `tfds` and a can provide conversion from one format to the other.",
        "\n",
        "To learn more about how to use metrics, take a look at the library 🤗 [Evaluate](https://huggingface.co/docs/evaluate/index)! In addition to metrics, you can find more tools for evaluating models and datasets."
```